src/main/java/org/parse4j/command/ParseDeleteCommand.java

```diff
@@ -9,9 +9,10 @@
 	private String endPoint;
 	private String objectId;
 
-	public ParseDeleteCommand(String endPoint, String objectId) {
+	public ParseDeleteCommand(String endPoint, String objectId, boolean useMasterKey) {
```

Comment:
What about add a method `useMasterKey` in `ParseCommand `
Then before calling `comman.perform()` we call `command.useMasterKey`
I think by doing this we'll minimize our changes, no?